.github/workflows/ci.yml

```diff
@@ -120,56 +120,14 @@ jobs:
         name: report-json
         path: .report.json
 
-    - name: Download reference test report (PR only)
+    - name: Upload test report (PR only)
```

Comment:
What's the difference between this step and the previous now? Do we need to make a distinction between PR and develop?

Reply:
I suppose not, no. it already exists only on the PR, I will remove.